src/backend.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-mutable-exports */
+let todos = [
+  { description: 'read es6 arrow functions', completed: false, index: 1 },
+  { description: 'read on dom events', completed: false, index: 2 },
```

Comment:
- [ ] **OPTIONAL**
Please notice that you repeated this code once in the data file. Instead, you could use import and export keywords for that matter 😊
_Kindly keep in mind that you'll learn and be asked about this point in the upcoming projects.👌🏻_